lib/Mojolicious/Plugin/OpenAPI/Guides/OpenAPI3.pod

```diff
@@ -0,0 +1,392 @@
+=head1 NAME
+
+Mojolicious::Plugin::OpenAPI::Guides::OpenAPI3 - Mojolicious <3 Open API (Swagger)
+
+=head1 OVERVIEW
+
+This guide will give you an introduction to how to use
+L<Mojolicious::Plugin::OpenAPI>.
+
+=head1 TUTORIAL
+
+=head2 Specification
+
+This plugin reads an L<OpenAPI specification|https://openapis.org/specification>
+and generate routes and input/output rules from it. See L<JSON::Validator> for
+L<supported schema file formats|JSON::Validator/Supported schema formats>.
+
+  {
+    "openapi": "3.0.2",
+    "info": {
+      "version": "1.0",
+      "title": "Some awesome API"
+    },
+    "paths": {
+      "/pets": {
+        "get": {
+          "operationId": "getPets",
+          "x-mojo-name": "get_pets",
+          "x-mojo-to": "pet#list",
+          "summary": "Finds pets in the system",
+          "parameters": [
+            {
+              "in": "query",
+              "name": "age",
+              "schema": {
+                "type": "integer"
+              }
+            }
+          ],
+          "requestBody": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
+              "description": "Pet response",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "pets": {
+                        "type": "array",
+                        "items": {
+                          "type": "object"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "servers": [
+      {
+        "url": "/api"
+      }
+    ]
+  }
+
+The complete HTTP request for getting the "pet list" will be C<GET /api/pets>
+The first part of the path ("/api") comes from C<basePath>, the second part
+comes from the keys under C<paths>, and the HTTP method comes from the keys
+under C</pets>.
+
+The different parts of the specification can also be retrieved as JSON using
+the "OPTIONS" HTTP method. Example:
```

Comment:
Does this work for v3 as well? (I haven't tested it, I think)

Reply:
I didn't actually notice, it kinda works. I will not place the references correctly, will wrongly tried to find "definitions". I will work on a PR for this.

Reply:
If Im not mistaken, part of the fix for this is in JSON::Validator